src/pages/log-management/opensearch/opensearch-dashboards/api.mdx

```diff
@@ -0,0 +1,86 @@
+---
+title: Access Opensearch Dashboard features via REST API
+metaTitle: Learn How to Access Opensearch Dashboards features via REST API
+description: Read Logit.io’s help article to learn how to access Opensearch Dashboards features via REST API.
+---
+
+# Accessing Opensearch Dashboards features via REST API
+
+You can use Basic Authentication to connect to the Logit.io Proxy and access Opensearch Dashboards without having to go 
+through the Logit.io Dashboard. This can improve your user experience and allow proper integration between your company, your applications 
+and the Logit.io service.
+
+You can, if you choose to, connect your application directly to Logit.io to do a range of things such as load dashboards, visualisations, 
+index patterns and mappings etc. or simply load them straight from your computer. 
+
+This article explains how to call REST API Endpoints to access Opensearch Dashboards using our Logit.io proxy and connect to it using Basic 
+Authentication.
+
+<Callout type="info">
+    Basic Authentication is a simple authentication method in HTTP(S) where the client includes a username and password with each request. 
+    The username and password is combined with a colon in-between as a string. You don't need to worry about how to create a basic authentication 
+    header as we have done it for you. 
+</Callout>
+
+Each request to a REST API Endpoint must include all of the necessary information for Opensearch Dashboards to fulfill the request. If all the necessary 
+information is provided the API requests return JSON output.
+
+Use the following operations to interact with the Opensearch Dashboards via the Logit.io Proxy.
+
+* __GET__ - Fetches the information.
+* __POST__ - Adds new information.
+* __PUT__ - Updates the existing information.
+* __PATCH__ - Applies partial modifications to the existing information.
+* __DELETE__ - Removes the information.
+
+All the details that you require in order to connect to the Logit.io proxy are found on the Profile Settings page of the Logit.io Dashboard. 
+The Profile Settings page has a section called `Proxy Settings` where you will find everything that you need. 
+Click [here](https://dashboard.logit.io/profile) to view you profile page.
+
+
+There are two ways that you can call the REST API using the Logit.io proxy. The call to our proxy needs to be in either of the following formats:
+
+__Basic Authentication header__:
+
+```bash
+curl -XGET "osd-xsrf:true" -H "Content-Type: application/json" 
+-H "Authorization: Basic {basic auth header}" {endpoint to call}
+```
+
+__Proxy Username and Password__:
+
+```bash
+curl -XGET "osd-xsrf:true" -H "Content-Type: application/json" 
+-u "{proxy username}:{proxy password}" {endpoint to call}
+```
```

Comment:
Do we need this and the examples below? Seems like an unnecessary duplication of code?

Reply:
I agree, I put this here when I thought I was writting a general article rather than an article for each usecase (opensearch dashboards, grafana, jaeger etc). I'll remove it.